frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -129,9 +133,17 @@ type cvdInstance struct {
 	Displays       []string `json:"displays"`
 	InstanceDir    string   `json:"instance_dir"`
 	WebRTCDeviceID string   `json:"webrtc_device_id"`
+	ADBSerial	   string   `json:"adb_serial"`
 }
 
-func (i *cvdInstance) toAPIObject(group string) *apiv1.CVD {
+func (i *cvdInstance) toAPIObject(group string) (*apiv1.CVD, error) {
+	host, port, err := net.SplitHostPort(i.ADBSerial)
+	if err != nil {
+		return nil, fmt.Errorf("failed to parse ADB serial: %w", err)
+	}
+	if host != "0.0.0.0" {
+		return nil, fmt.Errorf("invalid host address. (Expect: 0.0.0.0, Actual: %s)", host)
+	}
```

Comment:
Don't assert on the IP. This code doesn't seem to use it and it could start failing in the future if it changes to the external IP address, IPv6 or something else we haven't thought about.

Reply:
Removed the assertion. Done.